core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -465,21 +465,6 @@ public static function loadLanguageFile($strName, $strLanguage=null, $blnNoCache
 		// Use a global cache variable to support nested calls
 		static::$arrLanguageFiles[$strName][$strCacheKey] = $strLanguage;
 
-		// Backwards compatibility
-		if ('languages' === $strName)
-		{
-			// Reset previously loaded languages without destroying references
-			foreach (array_keys($GLOBALS['TL_LANG']['LNG'] ?? array()) as $strLocale)
-			{
-				$GLOBALS['TL_LANG']['LNG'][$strLocale] = null;
-			}
-
-			foreach (self::getContainer()->get('contao.intl.locales')->getLocales($strCacheKey) as $strLocale => $strLabel)
-			{
-				$GLOBALS['TL_LANG']['LNG'][$strLocale] = $strLabel;
-			}
-		}
-
```

Comment:
This currently makes the SystemTest#testLoadsLanguageTranslations()` test fail. But IMHO the test must be adjusted, removing the code here is fine.

Reply:
Done in 6eef9c516bc06f885e60466e2dd2323cb7abbe4d